sys/arvo.hoon

```diff
@@ -496,6 +506,7 @@
   ++  kick                                              ::  new main loop
     |=  {lac/? mor/(list muse)}
     =|  ova/(list ovum)
+    ~|  [%failed-kick lac]
```

Comment:
`lac` is the laconic bit, which is what gets flipped when you run `|verb`. I don't think it's worth printing. I think we should move this `~|` into `+jack', since that's the only part of this arm that could fail.